monitoring/utils/README.md

```diff
@@ -0,0 +1,61 @@
+# Utilities
+
+This directory contains utilities required to setup a monitoring infrastructure for application or generic metrics with chaos exporter metrics and litmus event router updates. 
+
+## Setups
+
+- Alert manager with service monitor
+
+  Contains setup for alert manager with its corresponding service monitor.
+  
+  ```
```

Comment:
Lets provide links or skip this block, multip `cd` can confuse the user

Reply:
Okay, I can provide the links.